.github/workflows/prepare-new-release.yml

```
@@ -34,116 +34,117 @@ jobs:
-    - name: Upload artifacts
-      run: gh release upload "$TAG_NAME" "$TMP_DIR"/*
-      env:
-        TMP_DIR: ${{ steps.random-dir.outputs.path }}
+
+      - name: Configure git
+        run: |
+          set -euo pipefail
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)